### PR TITLE
added link

### PR DESCRIPTION
**Pull request by RAJU2529**
added .net sdk 1.1   **https://dotnet.microsoft.com/download/dotnet/1.1**

### CODE REVIEW

**Comment by PRMerger15** [comment]
@RAJU2529 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: desktop-src/SecCrypto/setreg.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ The SetReg tool sets the value of the registry keys that control the behavior of
 
 **SetReg** \[*Options*\] \[*Choice \#* {**TRUE**\|**FALSE**}\]
 
-The Set Registry tool only ships with the .NET Framework SDK version 1.0 and .1.1, which you can download from the Microsoft Download Center.
+The Set Registry tool only ships with the .NET Framework SDK version 1.0 and .1.1, which you can download from the [here](https://dotnet.microsoft.com/download/dotnet/1.1).
```
Comment (alvinashcraft) [inline desktop-src/SecCrypto/setreg.md]: Can you change "from the [here]" to "from [here]"? Thanks!

Comment (alvinashcraft) [reply desktop-src/SecCrypto/setreg.md]:
```suggestion
The Set Registry tool only ships with the .NET Framework SDK version 1.0 and .1.1, which you can download from [here](https://dotnet.microsoft.com/download/dotnet/1.1).
```

**Comment by PRMerger17** [comment]
@RAJU2529 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by alvinashcraft — APPROVED** [review]
Looks good. Thanks again for your contribution.

File: desktop-src/SecCrypto/setreg.md (hunk 1 above)
Comment (RAJU2529) [reply desktop-src/SecCrypto/setreg.md]: @alvinashcraft , I have changed

Comment (alvinashcraft) [reply desktop-src/SecCrypto/setreg.md]: Got it! I will merge shortly.

**Pull request merged**